cli/podman.adoc

```diff
@@ -148,117 +191,147 @@ After you have linked to a network, you can check the link status:
 $ skupper link status
 ----
 
+include::link-cost.adoc[leveloffset=+1]
 
-== Working with services using Skupper podman
+// Type: concept
+[id="exposing-services-ns"] 
+== Exposing services on the {service-network} from a Linux host
+
+After creating a {service-network}, exposed services can communicate across that network.
 
 The general flow for working with services is the same for Kubernetes and Podman sites.
 
+The `skupper` CLI has two options for exposing services that already exist on a host:
 
-NOTE: Services exposed on Kubernetes are not automatically available to Podman sites.
-This is the equivalent to Kubernetes sites created using `skupper init --enable-service-sync false`.
+* `expose` supports simple use cases, for example, a deployment with a single service.
```

Comment:
wdyt of  `a host with a single service` (or a process) instead of `a deployment with a single service`?